File: C/Aula_Pratica_03/exec03.c
#include<stdio.h>
#include<stdlib.h>
#include<locale.h>

int main() {
	setlocale(LC_ALL, "");

    int numero;
	
	printf("Informe um número: ");
    fflush(stdin);
    scanf("%d", &numero);

    if (numero % 2 == 0) {
        printf("Este é um número PAR.");
    } else {
        printf("Este é um número ÍMPAR.");
    }
        
	system("pause");
    return 0;
}

File: Java/Aula_02/src/Exercicios/Exec03.java
package Exercicios;

import java.util.Scanner;

public class Exec03 {

	public static void main(String[] args) {
		
		Scanner teclado = new Scanner(System.in);
		
		int numero;
		
		System.out.println("Informe um número:");	
		numero = teclado.nextInt();
		System.out.println();
		
		switch(numero) {
//		case 1:
//		case 3:
//		case 5:
//		case 7:
//		case 9:
		case 1, 3, 5, 7, 9:
			System.out.println("Este número é ÍMPAR.");
			break;
		case 2:
		case 4:
		case 6:
		case 8:
		case 10:
			System.out.println("Este número é PAR.");
			break;
		default:
			System.out.println("Número inválido!");
	}
		
		teclado.close();
		
	}

}
